charts/consul/test/unit/connect-inject-deployment.bats

```diff
@@ -1883,19 +1883,64 @@ EOF
   [ "${actual}" = "false" ]
 }
 
-@test "connectInject/Deployment: -read-server-expose-service is not set when global.peering.tokenGeneration.serverAddresses.source is not equal to empty string" {
+@test "connectInject/Deployment: -read-server-expose-service and -server-address is not set when global.peering.tokenGeneration.serverAddresses.source is not equal to empty string" {
   cd `chart_dir`
-  local actual=$(helm template \
+  local command=$(helm template \
       -s templates/connect-inject-deployment.yaml  \
       --set 'connectInject.enabled=true' \
       --set 'global.peering.enabled=true' \
       --set 'global.peering.tokenGeneration.serverAddresses.source="notempty"' \
```

Comment:
not a blocker, but we should probably fail if you provide a value that we don't support

Reply:
Will add this in the last stacked PR, so we can fail on values that are not "static" or "consul" or ""

Reply:
added to the 3rd of the stacked PRs!